Source/Krypton Components/Krypton.Toolkit/Converters/PaletteButtonSpecStyleConverter.cs

```diff
@@ -44,7 +44,21 @@ internal class PaletteButtonSpecStyleConverter : StringLookupConverter
             new(PaletteButtonSpecStyle.WorkspaceMaximize, "Workspace Maximize"),
             new(PaletteButtonSpecStyle.WorkspaceRestore, "Workspace Restore"),
             new(PaletteButtonSpecStyle.RibbonMinimize, "Ribbon Minimize"),
-            new(PaletteButtonSpecStyle.RibbonExpand, "Ribbon Expand")
+            new(PaletteButtonSpecStyle.RibbonExpand, "Ribbon Expand"),
+            new(PaletteButtonSpecStyle.New, @"New"),
```

Comment:
These should be global strings to allow localisation

Reply:
Will do in a future update via KManager.Strings.